scripts/www/build-cfg-help.pl

```diff
@@ -247,9 +247,12 @@ ()
     }
 }
 
+my $linecount = 0;
+my @ifelse = {};
 while (<>) {
     chomp;
     last if (/^EOF$/);
+    $linecount++;
```

Comment:
If you insist on maintaining your own line count, please increment this line counter after reading a line, before any code that might exit the loop.

This is one of the two problems insist on addressing.